lib/rules/best-practises/no-unused-vars.js

```diff
@@ -71,14 +71,15 @@ class NoUnusedVarsChecker extends BaseChecker {
   }
 
   'FunctionDefinition:exit'(node) {
+    // we skip adding a var usage scope for function _declarations_ and
+    // functions with an empty / without a code block
```

Comment:
hard to parse as sth else than "functions with an empty slash that do not have a code block". Would rephrase

Reply:
resolve at will btw